spec/ddtrace/contrib/ethon/easy_patch_spec.rb

```diff
@@ -53,7 +53,7 @@
 
     it 'creates a span' do
       subject
-      expect(easy.instance_eval { @datadog_span }).to be_instance_of(Datadog::Span)
+      expect(easy.instance_eval { @datadog_span }).to be_instance_of(Datadog::SpanOperation)
     end
```

Comment:
Minor: Test description is inaccurate (this is no longer a span). I'm guessing this would apply to other specs, so it may be worth doing a bit of finding and replacing